LBH.AdultSocialCare.Api/V1/Gateways/DayCarePackageGateways/DayCarePackageGateway.cs

```diff
@@ -27,6 +30,22 @@ public async Task<Guid> CreateDayCarePackage(Infrastructure.Entities.DayCarePack
             return entry.Entity.DayCarePackageId;
         }
 
+        public async Task<DayCarePackageDomain> UpdateDayCarePackage(DayCarePackageForUpdateDomain dayCarePackageForUpdate)
+        {
+            var dayCarePackageEntity = await _dbContext.DayCarePackages
+                .Where(dc => dc.DayCarePackageId.Equals(dayCarePackageForUpdate.DayCarePackageId))
+                .SingleOrDefaultAsync().ConfigureAwait(false);
+            if (dayCarePackageEntity == null)
+            {
+                throw new ResourceNotFoundException($"Unable to locate day care package {dayCarePackageForUpdate.DayCarePackageId.ToString()}");
+            }
+            // Map fields with auto mapper and save
+            _mapper.Map(dayCarePackageForUpdate, dayCarePackageEntity);
+            dayCarePackageEntity.DateUpdated = DateTimeOffset.Now;
```

Comment:
The DateUpdate field can have `[DatabaseGenerated(DatabaseGeneratedOption.Computed)]` as an attribute, which will assign a value whenever inserted/updated, so we don't have to worry about manually providing a value

Reply:
> The DateUpdate field can have `[DatabaseGenerated(DatabaseGeneratedOption.Computed)]` as an attribute, which will assign a value whenever inserted/updated, so we don't have to worry about manually providing a value

Date created has computed. I leave date updated as null to only set it when the entity is updated